app/parse.py

```diff
@@ -15,8 +21,83 @@ class Product:
     num_of_reviews: int
 
 
-def get_all_products() -> None:
-    pass
+driver = webdriver.Chrome()
+wait = WebDriverWait(driver, 10)
+
+def accept_cookies():
+    try:
+        cookies_button = wait.until(EC.element_to_be_clickable((By.XPATH, "//button[text()='Accept']")))
+        cookies_button.click()
+        print("Accepted cookies.")
+    except:
```

Comment:
Using a bare `except` is not recommended as it catches all exceptions, including system-exiting exceptions. Consider catching specific exceptions like `NoSuchElementException` for better error handling.